.travis.yml

```diff
@@ -1,26 +1,48 @@
 language: scala
 sudo: false
+
+jdk:
+- oraclejdk8
+
 scala:
 - 2.12.6
 - 2.11.12
+
 install:
 - rvm use 2.4.4 --install --fuzzy
 - gem update --system
 - gem install sass
 - gem install jekyll -v 3.8.2
-jdk:
-- oraclejdk8
+
+stages:
+- name: test
+- name: release
+#  if: (branch = master AND type = push) OR (tag IS present)
+
+matrix:
+  include:
+  - env: TEST="scalafmt"
+    script: sbt ++$TRAVIS_SCALA_VERSION scalafmtCheck test:scalafmtCheck scalafmtSbtCheck
+  - env: TEST="test"
+    script: sbt ++$TRAVIS_SCALA_VERSION coverage test coverageReport microsite/makeMicrosite && bash <(curl -s https://codecov.io/bash)
+  - stage: release
+    script: sbt ci-release
```

Comment:
We need releases for both 2.11 and 2.12, `++$TRAVIS_SCALA_VERSION` should always be included